config/config.yaml

```diff
@@ -90,6 +90,7 @@ JobsDB:
       failedOnly: false
   gw:
     enableWriterQueue: false
+    maxOpenConnections: 64
```

Comment:
I would suggest having gw specific defaults in our code, this way is not necessary for this or devops config to be there for the system to work with reasonable configuration.

Reply:
I have started working on refactoring jobsdb initialisation & configuration, but as a separate task tracked [here](https://www.notion.so/rudderstacks/Refactor-jobsdb-intialisation-configuration-58e13e67e6194d0893fb8346ec38a4fa).